modules/quickstart-public-cloud/pages/setup.adoc

```diff
@@ -34,18 +23,20 @@ For security and access settings, you will need to enable HTTPS access.
 This allows you to access the {productname} {webui}.
 
 In EC2 and Azure, create a new security group, and add inbound and outbound rules for HTTPS.
-In GCE, check the ``Allow HTTPS traffic`` box under the ``Firewall`` section, .
+In GCE, check the ``Allow HTTPS traffic`` box under the ``Firewall`` section.
+
+[WARNING]
+====
+When running {productname} on public clouds, make sure you apply security measures to limit access to the right people.
+A world-accessible {productname} would give access to your infrastructure to anyone, and could breach your {suse} support agreement.
+====
 
 For storage settings, make sure you have at least 100GB.
 We also recommend that the repositories and the database for {productname} are stored on a separate storage volume.
```

Comment:
```suggestion
We also recommend that the repositories and the database for {productname} are stored on separate storage volumes.
```
Elsewhere you use plural.  And "devices" instead of "volumes".